src/pages/ClassComponent.mdx

```diff
@@ -176,3 +176,92 @@ export class ChildClassComponentWithObjectPropsMemoized extends Component<
 but when parent component changes `obj.str` re-rendering happens, as expected:
 
 <ObjectPropsClassComponent isMemoized={true} changeProps={true} />
+
+---
+
+### Regular Component - Method Component
+
+Method Component is a component which was implemented as a **method of Class Component**.
```

Comment:
Lets extract this to the separate page - Method Component
and add interactive example if it illustrates re-rendering